hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/contracts/execution/CallLocalEvmTxProcessorTest.java

```diff
@@ -71,6 +73,9 @@
 @ExtendWith(MockitoExtension.class)
 class CallLocalEvmTxProcessorTest {
     private static final int MAX_STACK_SIZE = 1024;
+    public static final long ONE_HBAR = 100_000_000L;
```

Comment:
nit: why do you need these constants?  will they be utilized later?

Reply:
TY. That was a copy-paste thing, needed for the mocks over at `CallEvmTxProcessorTest`, then some of those mocks weren't needed here because gas isn't even computed, and I forgot to clean up.